includes/class-sensei-utils.php

```diff
@@ -692,8 +692,8 @@ public static function sensei_delete_quiz_answers( $quiz_id = 0, $user_id = 0 ):
 		$quiz_submission = Sensei()->quiz_submission_repository->get( $quiz_id, $user_id );
 		if ( $quiz_submission ) {
 			Sensei()->quiz_submission_repository->delete( $quiz_submission );
-			Sensei()->quiz_answer_repository->delete_all( $quiz_submission->get_id() );
-			Sensei()->quiz_grade_repository->delete_all( $quiz_submission->get_id() );
+			Sensei()->quiz_answer_repository->delete_all( $quiz_submission );
+			Sensei()->quiz_grade_repository->delete_all( $quiz_submission );
```

Comment:
FYI (no action needed), deleting the answers and grades seems to not work when using tables (it works for comments). I've created a separate [issue](https://github.com/Automattic/sensei/issues/6493) to check that.